src/fr-webauthn/index.ts

```diff
@@ -141,13 +150,22 @@ abstract class FRWebAuthn {
         const credential = await this.getRegistrationCredential(publicKey);
         outcome = this.getRegistrationOutcome(credential);
       } catch (error) {
-        throw new Error(`WebAuthn: ${error.message}`);
+        // NotSupportedError is a special case
+        if (error.name === WebAuthnOutcomeType.NotSupportedError) {
```

Comment:
Looks good, just a tiny optional remark.
I would suggest renaming the 'NotSupportedError' to 'Unsupported' to be in sync with the WebAuthnOutcome naming.

Reply:
Hey @george-bafaloukas-forgerock , sorry I missed this. So, `NotSupportedError` is the official error from the spec, but AM is expecting us to map it to the `unsupported` outcome, and not the error outcome. I hope that make sense.